### DISCUSSION TITLE
Is support of text generative feature of Localai is available in kubernetes  environment ? if any performance testing performed in kubernetes environment

### DISCUSSION THREAD

**Discussion by harshsing2891** [mudler/LocalAI#1635] [Q&A]
**Is your feature request related to a problem? Please describe.**

**Describe alternatives you've considered**

**Additional context**

**Comment by lunamidori5** [comment]
Hello @harshsing2891 please fill out this post for this issue, thank you!

**Comment by harshsing2891** [comment]
**Is your feature request related to a problem? Please describe.**


**Describe alternatives you've considered**


**Additional context**

**Comment by harshsing2891** [comment]
comment updated
